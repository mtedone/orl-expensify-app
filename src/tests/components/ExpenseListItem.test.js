import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem with data', () => {
  const expense = expenses[0];
  const wrapper = shallow(<ExpenseListItem key={expense.id} {...expense} />);
  expect(wrapper.debug()).toMatchSnapshot();
});
