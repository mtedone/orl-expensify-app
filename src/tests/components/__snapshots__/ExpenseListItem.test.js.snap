// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with data 1`] = `
"<div>
  <Link to=\\"/edit/1\\">
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    $2.00
    -
    1st January 1970
  </p>
</div>"
`;
